sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -1276,8 +1279,10 @@ def get_module_fw_mgmt_feature(self, verbose = False):
                 txt += 'Read to LPL/EPL {:#x}\n'.format(rpl[6])
 
         else:
-            txt += 'Reply payload check code error\n'
-            return {'status': False, 'info': txt, 'result': None}
+            txt += 'Status or reply payload check code error\n'
+            logger.info(txt)
+            logger.info('Fail to get fw mgnt feature, cdb status: 0x{}\n'.format(status))
```

Comment:
```suggestion
            logger.info('Fail to get fw mgmt feature, cdb status: 0x{}\n'.format(status))
```

Reply:
@xinyulin Can you please log the value of `rpl_chkcode`?

Reply:
Sure. both rpl_chkcode and check result are recorded in the new commit.